pyproject.toml

```diff
@@ -1,33 +1,83 @@
 # For TOML reference
 # https://learnxinyminutes.com/docs/toml/
+[project]
+urls = { Documentation = "https://automl.github.io/DEHB/", Github = "https://github.com/automl/DEHB" }
+
+name = "DEHB"
+version = "0.7.0"
+dependencies = [
+  "numpy>=1.18.2",
+  "loguru>=0.5.3",
+  "dask>=2.27.0",
+  "distributed>=2.27.0",
+  "ConfigSpace>=0.4.16",
+]
+classifiers = [
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Natural Language :: English",
+  "Intended Audience :: Developers",
+  "Intended Audience :: Education",
+  "Intended Audience :: Science/Research",
+  "Topic :: Scientific/Engineering",
+  "Topic :: Scientific/Engineering :: Artificial Intelligence",
+]
+license = { file = "LICENSE.txt" }
+readme = "README.md"
+description = "Evolutionary Hyperband for Scalable, Robust and Efficient Hyperparameter Optimization"
+authors = [
+  { name = "Neeratyoy Mallik", email = "mallik@cs.uni-freiburg.de" },
+  { name = "Noor Awad" },
+  { name = "Frank Hutter" },
+  { name = "Janis Fix", email = "fixj@cs.uni-freiburg.de" },
+]
+requires-python = ">=3.8"
+[project.optional-dependencies]
+dev = [
+  # Test
+  "pytest>=4.6",
+  "pytest-cov",
+  "pytest-xdist",
+  "pytest-timeout",
+  # Docs
+  "mkdocs",
+  "mkdocs-material",
+  "mkdocstrings[python]",
+  "markdown-exec[ansi]",
+  # Others
+  "ruff",
+  "black",
+  "pre-commit",
+]
```

Comment:
Just some minor questions on things that have been specified in setup.py but have been left out here:
1. We do not specify the `package_name` property here, is this intended because it is not needed?
2. In `setup.py` we had a copyright field, is this also obsolete?
3. We had to specify, that the package is located in the `src` folder in order to be properly found when building. Is this not the case anymore?

Reply:
1. `pyproject.toml` by default follows the `src` style layout and infers things from that. It can be explicitly mentioned but I'm not sure how.
2. It was a field in the file but not used. In general this is covered by LICENSE which is mentioned.
3. Same as point 1.

The following works after a fresh install
```bash
python -c "import dehb"
```